prow/plugins/verify-owners/verify-owners_test.go

```diff
@@ -292,3 +294,134 @@ func TestHandle(t *testing.T) {
 		}
 	}
 }
+
+func TestParseOwnersFile(t *testing.T) {
+	cases := []struct {
+		name  string
+		patch string
+		err   bool
+		line  int
+	}{
+		{
+			name: "emptyApprovers",
+			err:  true,
+			line: 1,
+		},
+		{
+			name: "emptyApproversFilters",
+			err:  true,
+			line: 1,
+		},
+		{
+			name: "invalidSyntax",
+			err:  true,
+			line: 7,
+		},
+		{
+			name: "invalidSyntaxFilters",
+			err:  true,
+			line: 9,
+		},
+		{
+			name:  "invalidSyntaxFilters edit",
+			patch: "@@ -5,6 +5,6 @@ filters:\n     reviewers:\n     - alice\n     - bob\n-    labels\n+    label\n     - label1\n",
```

Comment:
Can we use grave accents (back-ticks) to format this nicely as a multiline string?